obal/__init__.py

```diff
@@ -7,6 +7,7 @@
 import os
 import sys
 
+import yaml
```

Comment:
Strictly speaking, this needs `PyYAML` as a dep in `setup.py` etc, right? (it gets pulled in by `ansible` anyways, but let's be explicit)